net.poedit.Poedit.json

```diff
@@ -1,21 +1,21 @@
 {
     "app-id": "net.poedit.Poedit",
     "runtime": "org.gnome.Platform",
-    "runtime-version": "3.26",
+    "runtime-version": "3.28",
```

Comment:
Why not 3.30?

Reply:
Build failed after update runtime to 3.30

```
...
configure: error: Package requirements (glib-2.0 gtk+-3.0 enchant) were not met:
Package 'enchant', required by 'virtual:world', not found
...
```

Reply:
You need to add enchant as it was removed from freedesktop 18.08

Reply:
Yeah, that should be done too, not sure if the app requires anything that isn't already on 18.08.

Reply:
Any idea to build `enchant`? 
Just take a look at https://gitlab.gnome.org/GNOME/recipes/blob/master/flatpak/org.gnome.Recipes.json and https://gitlab.gnome.org/GNOME/geary/blob/master/org.gnome.Geary.json, i have no clue.

Reply:
You have misclosed the wxwidgets module. You can just open the json file in any text editor with syntax highlithing to see the issue :+1: 
Thanks for taking care of updating it.